### PR TITLE
Run show page should have a link to its job

### PR DESCRIPTION
**Pull request by NavyAdmiral**
For [#165974907](https://www.pivotaltracker.com/story/show/165974907)

Don't know how to go about design/text for the link, this card looks okay-ish to me

asd

### CODE REVIEW

**Comment by rupurt** [comment]
@NavyAdmiral I feel like it should take up 100% width. I was originally thinking design wise something like we have on job#show.

Screen Shot 2019-05-15 at 9 04 42 AM

ping @thoughtmerchant?

**Comment by se3000** [comment]
Full width would be best. Also, it would be good to get the Job's ID in there, maybe as the link?

**Comment by NavyAdmiral** [comment]
image
@rupurt  What do you think about the attached picture? This card might take a bigger space than the one that you had imagined, but I think it's fine as long as there's no other component additions planned for this page

**Comment by thoughtmerchant** [comment]
image
Navigating from the job spec detail back to the job spec should be a breadcrumb above the H1 on the page. That makes a lot more sense from an IA standpoint. 

Having a link on the lower right that "goes up a level" breaks almost every UX heuristic. Take a look at the image attached and let me know your thoughts @NavyAdmiral @rupurt @se3000

**Comment by NavyAdmiral** [comment]
> image
> Navigating from the job spec detail back to the job spec should be a breadcrumb above the H1 on the page. That makes a lot more sense from an IA standpoint.
> 
> Having a link on the lower right that "goes up a level" breaks almost every UX heuristic. Take a look at the image attached and let me know your thoughts @NavyAdmiral @rupurt @se3000

I like that very much! Thanks for tuning in. 😅

**Comment by NavyAdmiral** [comment]
@thoughtmerchant Quick follow-up. Aren't `#`'s and `@`'s alone confusing experience? Should we prefix them with `Spec` and `Run`? 
image

**Comment by thoughtmerchant** [comment]
@NavyAdmiral trying to chip in when I can! @rupurt pings me on slack when it gets serious 😄 

We went back and forth on the prefixes but didn't come to a good consensus. The screenshot you are looking at is from an old design. I quickly copy pasta'd it into here for reference. 

Until our team, as a whole, decides on appropriate prefixing (there are a bunch of tradeoffs we have been discussing with this topic that are unresolved), I would just show the Spec ID for now as a clickable link. It looks like a breadcrumb and accomplishes the task for the user. That pattern should feel familiar with or without the prefix for the time being.

File: operator_ui/src/components/JobRuns/RegionalNav.js [hunk 1]
```diff
@@ -42,11 +42,16 @@ const styles = theme => {
...
 
-const RegionalNav = ({ classes, jobSpecId, jobRunId, jobRun }) => {
+const RegionalNav = ({ classes, jobSpecId, jobRunId }) => {
   return (
     <Card className={classes.container}>
       <Grid container spacing={0}>
         <Grid item xs={12}>
+          <Link to={`/jobs/${jobSpecId}`}>
+            <Typography variant="subtitle1" color="primary">
+              @{jobSpecId}
```
Comment (rupurt) [inline operator_ui/src/components/JobRuns/RegionalNav.js]: ✂️ `@`

**Pull request merged**